cloudify_agent/operations.py

```diff
@@ -168,9 +168,11 @@ def _save_daemon(daemon):
 
 def _set_default_new_agent_config_values(
         old_agent, new_agent, transfer_agent=False):
-    if not transfer_agent:
-        new_agent['name'] = utils.internal.generate_new_agent_name(
-            old_agent['name'])
+    if transfer_agent:
```

Comment:
tbh maybe it'd be better, instead of having a `transfer_agent` flag to `_set_default_new_agent_config_values` and `create_new_agent_config`, to have a new function (`create_transfer_agent_config`) that would call `create_new_agent_config`, and then overwrite the resulting agent with the transfer-specific changes? (ie. just the name i think)

Reply:
the problem with this is that `create_new_agent_config` is called from inside `_run_install_script`, so I'd have to change more than I want to